front/src/applications/stdcm/components/StdcmForm/StdcmDefaultCard.tsx

```diff
@@ -16,12 +16,12 @@ const StdcmDefaultCard = ({
   disabled = false,
   className = 'add-via',
 }: StdcmCardProps) => (
-  <StdcmCard hasTip={hasTip} disabled={disabled} className={className}>
-    <button type="button" onClick={onClick}>
+  <button type="button" onClick={onClick}>
```

Comment:
Otherwise the cards are still clickable when a simulation is retained

```suggestion
  <button type="button" disabled={disabled} onClick={onClick}>
```

Reply:
Good point, thanks for the hint 👍

Reply:
Done